stories/Divider.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Divider } from '../src'; // components you used in the story
+
+<Meta title="Layout/Divider" component={Divider} />
+
+# Divider
+
+Divider component serves as an hr for ease of templating. A very simple component 
+but can add a lot in the experience of the user. It can be used to seperate between sections.
+
+## Table of Contents
+
+In this documentation we talk about the Divider component in depth about its usage and its props.
+
+- [Functionality](#functionality)
```

Comment:
After you review the "Styling" subsection, I want you to add it to the Functionality section, you can do this by adding another bullet below functionality and making sure it is one `tab` more advanced.